Makefile

```diff
@@ -1289,6 +1298,116 @@ check_compile_tests
 .PHONY: conformance # Automatically fix problems that can be fixed
 conformance: fix_newline fmt
 
+#=============================== FFT Section ==================================
+.PHONY: doc_fft # Build rust doc for tfhe-fft
+doc_fft: install_rs_check_toolchain
+	@# Even though we are not in docs.rs, this allows to "just" build the doc
+	DOCS_RS=1 \
+	RUSTDOCFLAGS="--html-in-header katex-header.html" \
+	cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" doc \
+		--all-features --no-deps -p tfhe-fft
+
+.PHONY: docs_fft # Build rust doc tfhe-fft, alias for doc
+docs_fft: doc_fft
+
+.PHONY: lint_doc_fft # Build rust doc for tfhe-fft with linting enabled
+lint_doc_fft: install_rs_check_toolchain
+	@# Even though we are not in docs.rs, this allows to "just" build the doc
+	DOCS_RS=1 \
+	RUSTDOCFLAGS="--html-in-header katex-header.html -Dwarnings" \
+	cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" doc \
+		--all-features --no-deps -p tfhe-fft
+
+.PHONY: lint_docs_fft # Build rust doc for tfhe-fft with linting enabled, alias for lint_doc
+lint_docs_fft: lint_doc_fft
+
+.PHONY: clippy_fft # Run clippy lints on tfhe-fft
+clippy_fft: install_rs_check_toolchain
+	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy --all-targets \
+		--all-features -p tfhe-fft -- --no-deps -D warnings
+
+.PHONY: pcc_fft # pcc stands for pre commit checks
+pcc_fft: check_fmt lint_doc_fft clippy_fft
+
+.PHONY: build_fft
+build_fft: install_rs_build_toolchain
+	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release -p tfhe-fft
+	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release -p tfhe-fft \
+		--features=fft128
```

Comment:
Wouldn't the second build overwrite the first ?
I'm curious about the change of build command paradigm comparing to `concrete-fft`, could you explain a bit more?